bash/Afterburner_functionality.bash

```diff
@@ -92,11 +105,28 @@ function __static__Create_Sampled_Particles_List_File_Or_Symbolic_Link_With_Or_W
             "${HYBRID_software_input_file[Afterburner]}" \
             "${HYBRID_software_input_file[Spectators]}"
         # Run Python script to add spectators
-        "${HYBRID_external_python_scripts[Add_spectators_from_IC]}" \
-            '--sampled_particle_list' "${HYBRID_software_input_file[Afterburner]}" \
-            '--initial_particle_list' "${HYBRID_software_input_file[Spectators]}" \
-            '--output_file' "${target_link_name}" \
-            '--smash_config' "${HYBRID_software_output_directory[IC]}/config.yaml"
+        # NOTE: Since the errexit option enabled, the python script to add the spectators is run in the if-statement
+        #       and the possible exit code is accessed at the very beginning of the else-clause.
+        local python_exit_code
+        if __static__Run_Add_Spectators_Python_Script; then
+            # BE AWARE: The negation of the if-clause does not work because then the exit code cannot
+            #           be extracted properly (it will always be 0 because of the true if-statement).
+            :
+        else
+            python_exit_code=$?
+            if [[ ${python_exit_code} -eq ${HYBRID_fatal_value_error} ]]; then
+                exit_code=${HYBRID_fatal_value_error} Print_Fatal_And_Exit \
+                    'It was attempted to add spectators from multiple IC events to the sampled particles file. Only' \
+                    'running one IC event is supported when using the Afterburner config key ' \
+                    --emph "Add_spectators_from_IC: true" '.'
+            elif [[ ${python_exit_code} -eq 2 ]]; then
+                Print_Internal_And_Exit \
+                    'The Hybrid-handler variable ' --emph "HYBRID_fatal_value_error" ' does not seem to be set.'
```

Comment:
I think here we should point to the variable that the Python script tries to use.

```suggestion
                Print_Internal_And_Exit \
                    'The handing over of the ' --emph "python_fatal_value_error" \
                    ' to the Python script to add spectators did not work.'
```